config/spotbugs-exclude.xml

```diff
@@ -57,6 +57,16 @@
     <Method name="dumpMetadata" />
     <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE" />
   </Match>
+  <Match>
+    <!-- There is some empty methods in AbstractCheck/AbstractJavadocCheck.
+         Making them abstract requires too much boilerplate in subclasses. -->
+    <Bug pattern="ACEM_ABSTRACT_CLASS_EMPTY_METHODS"/>
+  </Match>
+  <Match>
+    <!-- Till https://github.com/mebigfatguy/fb-contrib/issues/296
+    @Nullable annotation is good idea in general, but requires extra dependencies. -->
```

Comment:
Very minor, but shouldn't this be indented like the item above it?

Reply:
done